### PR TITLE
MOTRECALLS 1012 create templates for recalls sms and email

### PR DESCRIPTION
**Pull request by alexisc-kainos**
## Description

New templates in Gov Notify for the recall messages that will be sent alongside MOT Reminders.

Related issue: https://dvsa.atlassian.net/browse/MOTRECALLS-1012

## Before submitting (or marking as "ready for review")

- [ ] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [ ] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: motr-notifier/src/main/resources/template/notify/outstanding-safety-recall-email-body.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+^#There's an outstanding safety recall on ((vehicle_details))
```
Comment (iichr) [inline motr-notifier/src/main/resources/template/notify/outstanding-safety-recall-email-body.txt:1]: I would imagine we'd use ((make)) ((model)) ((vrm)) here instead?

File: motr-notifier/src/main/resources/template/notify/outstanding-safety-recall-email-body.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+^# **Next steps:** Contact your nearest ((vehicle_make)) dealership for information and to arrange a free repair
+
+For more information visit: Check the MOT history of a vehicle
```
Comment (iichr) [inline motr-notifier/src/main/resources/template/notify/outstanding-safety-recall-email-body.txt:5]: URL instead as in the mockups? Might be examples in already existing templates if it needs inserted with a placeholder variable instead

File: motr-notifier/src/main/resources/template/notify/outstanding-safety-recall-email-body.txt (hunk 1 above)
Comment (alexisc-kainos) [reply motr-notifier/src/main/resources/template/notify/outstanding-safety-recall-email-body.txt:1]: I looked at this and I found, through the other templates, that vehicle_details is a string created of Make, Model and Vrm so it does the same thing

I can change it to take 3 explicit parameters though, if you like

Comment (iichr) [reply motr-notifier/src/main/resources/template/notify/outstanding-safety-recall-email-body.txt:1]: Oh interesting :) Leave as is then cool

**Review by StuartKainos — APPROVED** [review]

**Review by iichr — APPROVED** [review]

**Comment by alexisc-kainos** [comment]
Superseded by ticket 1015/1016

**Pull request closed without merging**